tests/src/FunctionalJavascript/Form/AuthenticationFormJsTest.php

```diff
@@ -20,7 +20,8 @@
 namespace Drupal\Tests\apigee_edge\FunctionalJavascript\Form;
 
 use Drupal\apigee_edge\Form\AuthenticationForm;
-use Drupal\apigee_edge\OauthTokenFileStorage;
+use Drupal\apigee_edge\Connector\OauthTokenFileStorage;
```

Comment:
You cannot move/rename classes because that is a BC break. You can create a new class in new locations and extend the new class in the old class and mark the old class as deprecated. 
Although, as the actual implementation did not change, I do not think this is necessary change.

Reply:
I've reverted this change.